tools/depends/target/webos-wayland-extensions/WEBOS-WAYLAND-EXTENSIONS-VERSION

```diff
@@ -0,0 +1,5 @@
+LIBNAME=webos-wayland-extensions
+BASE_URL=https://github.com/webosose/webos-wayland-extensions
```

Comment:
note: we'll need to upload those to our mirrors

Reply:
It used to be:

```
RETRIEVE_TOOL := curl -Ls --create-dirs --retry 10 --retry-delay 3
```

Which works for me. @lrusak should I add this back?

Reply:
@sundermann now fixed

Reply:
It'll need to be uploaded to the Kodi mirrors anyway, not sure who can do it. Perhaps @fuzzard ?

Reply:
I've uploaded v43 and v45 to mirrors:
```
# sha512sum webos-wayland-extensions-4*
5f89d2ef9dbfcb952900e691871da704e6ca9169a531bf308354800d6ee6395dcee33d1961df2861f3646f849ce92b38f352663475cab7512134dd3a91373181  webos-wayland-extensions-43.tar.gz
eca01641ddc9d4300e584da511a4d677f9330eea79206011a535e67dfa87fe755f777035407be468106628af830fb330030b930a679f1013057bce2d3f7cf3fc  webos-wayland-extensions-45.tar.gz

```

Reply:
to switch to the version from our mirrors, you have to delete BASE_URL